libtransmission/peer-mgr.cc

```diff
@@ -314,17 +340,20 @@ class tr_swarm
 
     [[nodiscard]] tr_peer_info* get_existing_peer_info(tr_socket_address const& socket_address) noexcept
     {
-        auto&& it = pool.find(socket_address);
-        return it != pool.end() ? &it->second : nullptr;
+        auto&& it = known_connectable.find(socket_address);
```

Comment:
Welp SEGV here... How did that happen...

Reply:
Looks like 5ce3c7e got it. Took longer than I like, but am glad I managed to catch a few other time bombs waiting to explode on the way.